torch/csrc/utils/tensor_new.cpp

```diff
@@ -215,41 +215,16 @@ void recursive_store(
   bool is_symfloat = torch::is_symfloat(obj);
   bool is_symint = torch::is_symint(obj);
   if (dim == ndim) {
+    auto new_obj = py::reinterpret_borrow<py::object>(obj);
```

Comment:
Why does this need to be moved out of the conditional?

Reply:
Fixed.